### DISCUSSION TITLE
ValidationError: use `allowGlobalContext` question

### DISCUSSION THREAD

**Discussion by samuraitruong** [mikro-orm/mikro-orm#3108] [Q&A]
**Describe the bug**

ValidationError: Using global EntityManager instance methods for context specific actions is disallowed. If you need to work with the global instance's identity map, use `allowGlobalContext` configuration option or `fork()` instead.
- we develop using nestJS the code is working fine on local development but on our deploy (prod) environment running on ECS + RDS (postgress) we get above error.

I did pull the image to local and run again with my local database (docker) and can not produce the error. 

**Stack trace**
```
...
```

**To Reproduce**
NA

**Expected behavior**
I would like to understand why this issue happen, will it cause by database?
will there any chance that make _id = 1 and cause the issue?

**Additional context**

**Versions**

| Dependency | Version |
| - | - |
| node | 16x |
| typescript | 4.6.4 |
| mikro-orm | 5.1.13 |
| your-driver | postgress |

**Comment by B4nan** [comment]
Apparently something is breaking the async context. Can't help with such issues given you did not provide any reproduction. Note that this points to a bug elsewhere, not in the ORM. This has nothing to do with the database.

As an alternative you could use the nest DI request scopes.

https://github.com/mikro-orm/nestjs#using-nestjs-injection-scopes-for-request-context
